impeller/display_list/aiks_dl_runtime_effect_unittests.cc

```diff
@@ -0,0 +1,87 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <memory>
+
+#include "display_list/effects/dl_color_source.h"
+#include "flutter/impeller/aiks/aiks_unittests.h"
+
+#include "flutter/display_list/dl_builder.h"
+#include "flutter/display_list/dl_paint.h"
+
+#include "include/core/SkPath.h"
+#include "include/core/SkRRect.h"
+
+namespace impeller {
+namespace testing {
+
+using namespace flutter;
+
+namespace {
+std::shared_ptr<DlRuntimeEffectColorSource> MakeRuntimeEffect(
+    AiksTest* test,
+    std::string_view name,
+    const std::shared_ptr<std::vector<uint8_t>>& uniform_data = {},
+    const std::vector<std::shared_ptr<DlColorSource>>& samplers = {}) {
+  auto runtime_stages =
+      test->OpenAssetAsRuntimeStage("runtime_stage_example.frag.iplr");
+
+  auto runtime_stage = runtime_stages[PlaygroundBackendToRuntimeStageBackend(
+      test->GetBackend())];
+
```

Comment:
Is it worth copying over the asserts from the original code?

Reply:
Done